poveste.txt

```diff
@@ -0,0 +1,13 @@
+Eram adunați într-un mănunchi de cântec mut. Fără sunet, vocile noastre se sublimau sub tensiunea care ne strângea tâmplele, străpungeau cu greu membranele individuale și se adunau undeva deasupra într-un nor de tensiune. 
```

Comment:
Are you sure ca suna bine « tensiune » de 2 ori aici ? E vb de ac tensiune sau sunt tensiuni diferite?